cmake/cmake.define

```diff
@@ -172,21 +172,19 @@ ELSE ()
         SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -msse4.2")
     ENDIF()
 
-    IF ("${SIMD_SUPPORT}" MATCHES "true")
```

Comment:
should check the simd_support macro to avoid generating the implicit fma instructions by compiler.

Reply:
Resolved.